docs/reference/glossary.rst

```diff
@@ -127,6 +127,14 @@ Glossary
       RBNO is enabled by default for a subset node operations. 
       See :doc:`Repair Based Node Operations </operating-scylla/procedures/cluster-management/repair-based-node-operation>` for details.
 
+    RF-rack-invalid keyspace
+      A keyspace is RF-rack-invalid if and only if it uses :doc:`tablet-based distribution </architecture/tablets>` and
+      there exists a data center for which the specified replication factor is not equal to: 0, 1, or the number of racks
+      in that data center.
+
+    RF-rack-valid keyspace
+      A keyspace is RF-rack-valid if and only if it's not RF-rack-invalid.
```

Comment:
Maybe that appears to the mathematically inclined, but a definition with fewer indirections and double negatives is better for most people.